lib/taskclusterrc.js

```diff
@@ -0,0 +1,174 @@
+var debug   = require('debug')('github:worker');
+var yaml    = require('yaml-js');
+var Promise = require('promise');
+var slugid  = require('slugid');
+var _       = require('lodash');
+
+var taskclusterrc = module.exports = {};
+
+/* Builds a url pointing to the raw path of some file in a github
+ * repository, when given an object containing fields for a github
+ * organization, repository, and branch.
+ */
+taskclusterrc.buildConfigUrl = function(params) {
+  return [
+    "https://raw.githubusercontent.com",
+    params.organization,
+    params.repository,
+    params.details.branch,
+    ".taskclusterrc"
+  ].join("/");
+}
+
+/**
+ * The minimum viable task graph configuration.
+ **/
+function buildGraphSkeleton() {
+  return {
+    tasks: [],
+    scopes: [
+      'queue:*',
+      'docker-worker:*',
+      'scheduler:*'
+      ],
+    metadata: {
+     owner: 'taskcluster-github@mozilla.com',
+     name: 'taskcluster-github graph',
+     description: 'A task graph submitted by the taskcluster-github worker',
+     source: 'https://github.com/taskcluster/taskcluster-github'
+    }
+  };
+};
+
+/**
+ * The minimum viable task configuration.
+ **/
+function buildTaskSkeleton() {
+  let defaultCreated = new Date();
+  let defaultDeadline = new Date(defaultCreated);
+  defaultDeadline.setHours(defaultDeadline.getHours() + 24);
+  return {
+    provisionerId: 'aws-provisioner-v1',
+    workerType: 'b2gtest',
+    retries: 5,
+    metadata: {
+     owner: 'taskcluster-github@mozilla.com',
```

Comment:
Owner is the person who created the PR or the push.
Github won't supply emails for all users, but a lot of user have a valid email we should supply that...
If there is no valid email, we should use same pattern as github: `<username>@users.noreply.github.com`

See: https://help.github.com/articles/keeping-your-email-address-private/

Note: whatever string must be a valid email format..
JSON schema will validate this! Later we may use this to allow persona authed users to cancel/rerun their own tasks.. Or we use it for stats... Either way we should strive to have something that identifies the person. And if not a real email, then a consistent pattern used elsewhere too.